en/frames_multicopter/holybro_x500v2_pixhawk6c.md

```diff
@@ -0,0 +1,159 @@
+# Holybro X500 V2 + Pixhawk 6C (PX4 Dev Kit)
+
+This topic provides full instructions for building the [Holybro X500 V2 ARF Kit](http://shop.holybro.com/x500-v2-kit_p1288.html), also known as the Holybro PX4 Dev Kit.
+
+![The fully built vehicle with props removed](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/kit_no_props.jpg)
+
+
+## Assembly
+
+### Holders
+
+**Screw**-  Sunk Screw M2.5*6 12pcs
```

Comment:
What does this mean? Is this the stuff you need to do this section? Aren't there other things you need here?

Reply:
> What does this mean? Is this the stuff you need to do this section? Aren't there other things you need here?

That's at the beginning of every section to mention which screws would be required ahead.